packages/asset-swapper/contracts/src/AaveV2Sampler.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2021 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "./SamplerUtils.sol";
+
+contract AaveV2Sampler is SamplerUtils {
+
+    struct AaveInfo {
+        address lendingPool;
+        address aToken;
+        address underlyingToken;
+    }
+
+    function sampleSellsFromAaveV2(
+        AaveInfo memory aaveInfo,
+        address takerToken,
+        address makerToken,
+        uint256[] memory takerTokenAmounts
+    )
+        public
+        pure
```

Comment:
Ideally we would just do all this in AS and skip the eth_call but I can tend to that.

Reply:
Good point! This file entirely unnecessary, move this logic into AS instead 🙏

Reply:
Moved to local sampling in f26497f815f306c211685d66bf348ef164b529df 👍